### ISSUE TITLE
IPHost REQUIRED_FOR_PUT needs to be updated

### ISSUE THREAD

**Issue by marksull** [daxm/fmcapi#25]
IPHost Is currently using the super default of APIClassTemplate.REQUIRED_FOR_PUT which is ['id'] but should be set to:

```
class IPHost(APIClassTemplate):
    REQUIRED_FOR_PUT = ['id', 'name', 'value']
```

**Comment by daxm** [comment]
Can you fix and issue a pull request?

**Comment by marksull** [comment]
Definitely will. As I get to the end of my current sprint I’ll come back through and fix anything that I find but I will create issues so I can track them.

**Comment by daxm** [comment]
Sweet!  Thanks.

**Comment by marksull** [comment]
Resolved with #26
